kafka-connect-s3/src/test/java/io/confluent/connect/s3/S3SinkConnectorConfigTest.java

```diff
@@ -409,5 +410,24 @@ public void testUnsupportedParquetCompressionType() {
     connectorConfig = new S3SinkConnectorConfig(properties);
     connectorConfig.parquetCompressionCodecName();
   }
+
+  @Test
+  public void testS3StorageClassSupported() {
+    properties.put(S3SinkConnectorConfig.S3_STORAGE_CLASS, "STANDARD");
```

Comment:
No need to re-type these

```suggestion
    properties.put(S3SinkConnectorConfig.S3_STORAGE_CLASS, StorageClass.Standard.toString());
```